src/core/lib/security/transport/security_connector.c

```diff
@@ -509,7 +556,8 @@ static grpc_error *ssl_check_peer(grpc_security_connector *sc,
 static void ssl_channel_check_peer(grpc_exec_ctx *exec_ctx,
```

Comment:
I'm not sure I understand. Are you suggesting adding check to the ssl security connector impl?